docs/how-to-guides/config-settings.rst

```diff
@@ -9,41 +9,53 @@ Use build config settings
 *************************
 
 Build config settings are settings you can pass to ``meson-python`` when
-building your project. They can be use to customize the build in some aspect.
+building your project. They can be use to customize the build in some
+aspect.
 
-The way you specify them depends on which tool you are using, here are the most
-popular ones.
+Several Python build front-ends exist, with different ways to pass
+configuration settings to the build back-end. The most popular are
+``pypa/build``, which uses the ``-C`` command line option, and ``pip``,
```

Comment:
- It is the style the docs are currently following
- Gives users a clickable link to the project, which is relevant if they are using pip for eg. and didn't know about pypa/build
- There's no downside?

> Again, without motivating your objections, it seems that you are disagreeing for the sake of disagreeing

I am not trying to disagree, these are just suggestions. I spent a significant amount of time going over and reviewing this PR, which I did not have to do, I could simply have a high level look and said :+1:, but I care about the docs, so I took the time to note what I consider to be possible improvements.

Isn't this common? When I open PRs, at least, I expect there to be comments, nothing I write is perfect or beyond improvement, and all that is subjective anyway. The point of reviews is to get other people's perspective, and see how it can be improved.

Anyway, I don't think our communication is very healthy at the moment. Is there anything you think we can do to try to improve this?
I sent you an email a couple days ago, were you able to read it over?

Reply:
I expect feedback to come with explanation. Otherwise it seems that your opinion is more important just because it is your opinion. I think it is the fundamental rule of providing feedback: if you cannot motivate it, it is just a persona preference, and there is no reason why your personal preference should be more important than the preference of someone else.

I don't think there is a communication issue. I disagree with you sometimes and you don't seem to handle that very well. See for example the interaction we just had on #360. I don't think that what you wrote is the proper way to deal with disagreement on technical matters. In particular the "Tired" comment you used to retire your objections. It makes it look like you retired them only because I've worn you out repeating the same arguments over and over. Instead the issue was already discussed in #306 where you decided to do not contribute to the discussion and the decision settled. I tried to provide the reasoning motivating the outcome of the discussion and you, instead of replying in the merit, decided to act in IMHO an immature way and declare that you don't care and I can have it may way if I really insist. If you really don't care, why did you object in the first place?

> I care about the docs, so I took the time to note what I consider to be possible improvements

How did the document I'm amending find its way in the documentation then?